CSVZipExport/module.php

```diff
@@ -60,7 +60,12 @@ public function GetConfigurationForm()
             'minute' => 59,
             'second' => 59
         ];
-        $jsonForm['elements'][1]['options'] = $this->GetOptions();
+        if (IPS_ModuleExists('{7E122824-E4D6-4FF8-8AA1-2B7BB36D5EC9}')) {
```

Comment:
Der Kommentar aus dem anderen Modul war toll. Den hätte ich hier auch gerne:

`//If the module "SyncMySQL" is install, get other options`